lib/src/typewriter.dart

```diff
@@ -61,8 +66,18 @@ class TypewriterAnimatedText extends AnimatedText {
   /// Widget showing partial text
   @override
   Widget animatedBuilder(BuildContext context, Widget child) {
+    /// Output of CurveTween is in the range [0, 1] for majority of the curves.
+    /// It is converted to [0, textCharacters.length].
     final textLen = textCharacters.length;
-    final typewriterValue = _typewriterText.value;
+    var typewriterValue =
+        (_typewriterText.value * textCharacters.length).round();
```

Comment:
Your change is ignoring the `extraLengthForBlinks`.  The expression should be:
```dart
(_typewriterText.value * (text.characters.length + extraLengthForBlinks)).round(),
```

Don't forget to update the comment too.

Reply:
Sure, I shall correct that.